plugins/redfish/fu-plugin-redfish.c

```diff
@@ -276,6 +276,9 @@ fu_redfish_plugin_ipmi_create_user(FuPlugin *plugin, GError **error)
 		return FALSE;
 	if (!fu_plugin_set_config_value(plugin, "Password", password_new, error))
 		return FALSE;
+	if (!fu_plugin_set_secure_config_file(plugin, error))
```

Comment:
Maybe we want a `fu_plugin_set_config_value_secure()` that does this atomically? It seems kinda risky to have a window of time where the file is readable by non-root with a password in.

Reply:
That's true.  Will swap around.